applications/nrf5340_audio/src/modules/nrf5340_dk.h

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#ifndef __BOARD_H__
-#define __BOARD_H__
+#ifndef __NRF5340_DK__
```

Comment:
There is something with naming. Everything in this file is related to the Audio DK

Reply:
Yes at the moment, but will we have to have a nRF54 file with different names?
We'd have to decide with some mechanism in the calling code, rather than just including a different header to the included data. Ideally the audio_nrf5340_dk_version.c file would have a better, more generic name, but that wasn't allowed. This could then be a library that will get the info for the relevant DK or return NULL.

Reply:
After discussion this file has been renamed nrf5340_audio_dk.h as only used for the audio DK.